### PR TITLE
Handle conv layer spec when input0 is type SpaceToBatchND

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

File: tests/test_layers.py [hunk 1]
```diff
@@ -193,15 +193,16 @@ def my_func_2(x):
...
-    model = Sequential()
-    model.add(InputLayer(input_shape=(8, 8, 2)))
-    model.add(keras.layers.DepthwiseConv2D(
-        kernel_size=(3, 3), strides=(1, 1), padding="VALID",
-        data_format='channels_last'))
-    onnx_model = keras2onnx.convert_keras(model, 'test_depthwise_conv2d')
-    data = np.random.rand(3, 8, 8, 2).astype(np.float32)
-    expected = model.predict(data)
-    assert runner('onnx_depthwise_conv2d', onnx_model, data, expected)
+    for use_bias_ in [True, False]:
```
Comment (wenbingl) [inline tests/test_layers.py:196]: should it be the test argument?

File: keras2onnx/_builtin.py [hunk 2]
```diff
@@ -118,43 +118,45 @@ def convert_tf_batch_to_space(scope, operator, container):
                                       starts=[0], ends=[1])
         block_prod = oopb.apply_mul(block_size + block_size,
                                     name=operator.full_name + '_mul_0')
-        padded_block_prod = oopb.apply_pad([block_prod,
-                                            ('const_zero_three', oopb.int64, np.array([0, 3], dtype='int64')),
-                                            ('one', oopb.int64, np.array([1], dtype='int64'))],
-                                           name=operator.full_name + '_pad_0')
+        padded_block_prod = oopb.apply_pad(block_prod,
+                                           name=operator.full_name + '_pad_0',
+                                           pads=[0, 3],
```
Comment (wenbingl) [inline keras2onnx/_builtin.py:123]: where does pads come from?

Comment (jiafatom) [reply keras2onnx/_builtin.py:123]: This is used for division, it is the dimension of the channel, so 0 means N, 3 means C

File: tests/test_layers.py (hunk 1 above)
Comment (jiafatom) [reply tests/test_layers.py:197]: done

**Pull request merged**